src/js/items/Component.js

```diff
@@ -31,7 +31,10 @@ lm.utils.copy( lm.items.Component.prototype, {
 	},
 
 	setSize: function() {
-		this.container._$setSize( this.element.width(), this.element.height() );
+		if( this.element.is( ':visible' ) ) {
+			// Do not update size of hidden components to prevent unwanted reflows
+			this.container._$setSize( this.element.width(), this.element.height() );
```

Comment:
Do you happen to know what might happen when a hidden item becomes visible again? Does it get re-`_$setSize`d on being made visible?